src/client/containers/batch-detail-page/batch-detail-page.component.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import SidebarMenu from '../../components/side-navigation/sidebar.component';
+import AddBatch from '../../components/batch/add-batch.component';
+import EditBatchImage from '../../components/batch/edit-batch-image/edit-batch-image.component';
```

Comment:
link should be '../../components/edit-batch-image/edit-batch-image.component'
```suggestion
import EditBatchImage from '../../components/edit-batch-image/edit-batch-image.component';
```